contrib/fuzz/archive_fuzzer.go

```diff
@@ -73,6 +77,31 @@ func FuzzImportIndex(data []byte) int {
 	}
 	ctx := context.Background()
 	r := bytes.NewReader(tarBytes)
+	doThis, err := f.GetBool()
+	if err != nil {
+		return 0
+	}
+	if doThis {
+		tr := tar.NewReader(r)
+		for {
+			hdr, err := tr.Next()
+			if err == io.EOF {
+				break
+			}
+			if err != nil {
+				return 0
+			}
+			hdrName := path.Clean(hdr.Name)
+			switch hdrName {
+			case ocispec.ImageLayoutFile:
+				break
+			case "manifest.json":
+				break
+			default:
+				return 0
```

Comment:
Since the tar is generated randomly (?), what's the likelihood that the tar will include a layout or manifest file?

Reply:
The likelihood is high once the fuzzer realises that doing so will uncover more coverage.

Reply:
The purpose of this PR is to signal to the fuzzer that creating a layout or a manifest file will uncover more coverage.

Reply:
Thanks, didn't understand that's how the fuzzer worked.